src/modules/enrollment/components/dashboardPages/EnrollmentCustomCaseNotesPage.tsx

```diff
@@ -0,0 +1,158 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Box, Button } from '@mui/material';
+import { useCallback, useState } from 'react';
+
+import { ColumnDef } from '@/components/elements/table/types';
+import TitleCard from '@/components/elements/TitleCard';
+import { useEnrollmentDashboardContext } from '@/components/pages/EnrollmentDashboard';
+import NotFound from '@/components/pages/NotFound';
+import DeleteMutationButton from '@/modules/dataFetching/components/DeleteMutationButton';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import { useFormDialog } from '@/modules/form/hooks/useFormDialog';
+import useViewDialog from '@/modules/form/hooks/useViewDialog';
+import { parseAndFormatDateTime } from '@/modules/hmis/hmisUtil';
+import { cache } from '@/providers/apolloClient';
+import {
+  CustomCaseNoteFieldsFragment,
+  DeleteCustomCaseNoteDocument,
+  DeleteCustomCaseNoteMutation,
+  DeleteCustomCaseNoteMutationVariables,
+  FormRole,
+  GetEnrollmentCustomCaseNotesDocument,
+  GetEnrollmentCustomCaseNotesQuery,
+  GetEnrollmentCustomCaseNotesQueryVariables,
+} from '@/types/gqlTypes';
+
+const columns: ColumnDef<CustomCaseNoteFieldsFragment>[] = [
+  {
+    header: 'Content',
+    render: ({ content }) => (
+      <Box sx={{ whiteSpace: 'pre-wrap' }}>{content}</Box>
+    ),
+  },
+  {
+    header: 'Created by',
+    width: '300px',
```

Comment:
small thing, but it could be `minWidth`. Looks kinda funny if all the notes are short on a wide screen.

Reply:
Sure, I'll fiddle with it. The notes are a weird shape for a table tbh, I would think cards would be a better fit but maybe we will need additional columns that require a table

Reply:
yeah..I think just going with what we have, for now. I tried `minWidth` and it was fine. Also totally fine to leave as-is.